R/DESCRIPTION

```diff
@@ -33,8 +33,10 @@ Collate:
     'flags.R'
     'flow.R'
     'metaflow_client.R'
-    'metaflow.R'
+    'metaflow-package.R'
```

Comment:
Any specific reason to change the name? Also, for consistency - `metaflow_package` could be a better choice.

Reply:
Nope not necessary, other than it's convention to either use `____-package.R` or `package.R`.

- https://cran.r-project.org/web/packages/roxygen2/vignettes/rd.html
- `usethis::use_package_doc` creates a file with the `____-package`

Sorry about these opinionated changes in this PR, was on autopilot for this one. It's totally out of scope of the business logic being changed here so I can leave these modifications out.

Reply:
Can we change it to `package.R`?